src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const isWoman = person.sex === 'f';
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={cn({ 'has-text-danger': isWoman })}
```

Comment:
The class 'has-text-danger' is applied conditionally here based on gender. Consider documenting this behavior to clarify why the color coding is important, as it may not be immediately obvious to other developers or users.